modules/criteoBidAdapter.js

```diff
@@ -24,7 +24,7 @@ const LOG_PREFIX = 'Criteo: ';
   Unminified source code can be found in the privately shared repo: https://github.com/Prebid-org/prebid-js-external-js-criteo/blob/master/dist/prod.js
 */
 const FAST_BID_VERSION_PLACEHOLDER = '%FAST_BID_VERSION%';
-export const FAST_BID_VERSION_CURRENT = 117;
+export const FAST_BID_VERSION_CURRENT = 123;
```

Comment:
The repo in the line above stops at 116 currently

Reply:
Indeed it looks like the updates were not uploaded there
We've fixed it, thanks for reporting it Patrick